### DISCUSSION TITLE
The "cache-control" header is present but performance testing tools return errors about it

### DISCUSSION THREAD

**Discussion by johan-Rm** [nuxt/nuxt#15975] [General]
I'm about to power a project under Nuxt 3 https://preprod.kazengarden.com

I try to optimize the application, when I do a test with https://pagespeed.web.dev/ I receive "Serve static assets with an efficient cache policy" but I see with CURL that the "cache-control" header is present.

----------------------------------------
```shell
$ curl -I https://preprod.kazengarden.com/_nuxt/entry.8fe1d28c.js
```

HTTP/2 500
date: ven. 09 déc. 2022 16:08:12 GMT
serveur : Apache/2.4.25 (Debian)
strict-transport-security : max-age=31536000 ; inclure les sous-domaines
cache-control : maxage=86400 s-maxage=86400
type de contenu : application/json
etag : "610eb-Ai+ZkCpqUvgBG9CeJ6+xWH9XrzU"
dernière modification : 2022-12-09T14:54:37.358Z
longueur du contenu : 397547
synchronisation du serveur : -;dur=0;desc="Générer"

I am a taker of all light

**Comment by narasimhajupally** [comment]
I think it should be `cache-control: max-age=300, s-maxage=300`
the comma is missing
